shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1973,6 +1976,23 @@ void WebContents::LoadURL(const GURL& url,
   NotifyUserActivation();
 }
 
+// TODO(MarshallOfSound): Figure out what we need to do with post data here, I
+// believe the default behavior when we pass "true" is to phone out to the
+// delegate and then the controller expects this method to be called again with
+// "false" if the user approves the reload.  For now this would result in
+// ".reload()" calls on POST data domains failing silently.  Passing false would
+// result in them succeeding, but reposting which although more correct could be
+// considering a breaking change.
```

Comment:
should we expose this as a param?

Reply:
I think we should, but that also (for correct implementations) would require us exposing the delegate event `did-get-post-data-reload-request` or something 🤔  I think for now defaulting to the safe value and exposing the new API later is OK.

Thoughts?

Reply:
sgtm